### ISSUE TITLE
maximizeWindow breaks tests on MacOS

### ISSUE THREAD

**Issue by cmbirk** [DevExpress/testcafe#1724]
### Are you requesting a feature or reporting a bug?

Bug

### What is the current behavior?

maximizeWindow breaks tests on MacOS.  The tests stop running, and the title is set to a series of odd characters: `[ BJ761BFOZ ]` that differ per run.  The tests do not fail, they simply stop executing anything

### What is the expected behavior?

Window should be maximized

### How would you reproduce the current behavior (if this is a bug)?

Simply call maximizeWindow on any of my tests

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```js
 await t
    .maximizeWindow()
    .expect(switchLoginButton.exists)
    .ok()
```

### Specify your

* operating system: MacOS 10.12.6
* testcafe version: 0.17.1
* node.js version: v6.9.5
* browser version: Chrome 60.0.3112.101

**Comment by AndreyBelym** [comment]
Hello @cmbirk! `maximizeWindow` uses OS functions to maximize windows, so TestCafe sets `document.title` to a unique ID (like `[ BJ761BFOZ ]`) to be able to find the browser in window in all other opened windows. So a title like this is normal.

There are third-party applications that can significantly delay browser window searching. Usually logging out without saving the session helps in this situation, did you try it?

**Comment by AndreyBelym** [comment]
Hi @cmbirk, the issue should be fixed by https://github.com/DevExpress/testcafe-browser-tools/pull/132, which is merged in current stable version `0.18.0`! So i'm closing it, but feel free to reopen if you still have the problem.

**Comment by cmbirk** [comment]
Thanks @AndreyBelym !!
